src/components/app/Home.js

```diff
@@ -351,15 +370,9 @@ class Home extends PureComponent {
             Capture a performance profile. Analyze it. Share it. Make the web
             faster.
           </p>
-          <CSSTransitionGroup
-            transitionName="homeTransition"
-            transitionEnterTimeout={300}
-            transitionLeave={false}
-            component="div"
-            className="homeInstructionsTransitionGroup"
```

Comment:
most of these goes to the `<CSStransition>` tag instead.
Note that `div` is the default value for `component` so I left it out.